tools/SourceKit/tools/sourcekitd/lib/API/sourcekitdAPI-InProc.cpp

```diff
@@ -0,0 +1,941 @@
+//===--- sourcekitdAPI-InProc.cpp -----------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#include "DictionaryKeys.h"
+#include "sourcekitd/sourcekitd.h"
+#include "sourcekitd/Internal.h"
+#include "sourcekitd/CodeCompletionResultsArray.h"
+#include "sourcekitd/DocSupportAnnotationArray.h"
+#include "sourcekitd/TokenAnnotationsArray.h"
+#include "sourcekitd/Logging.h"
+#include "SourceKit/Core/LLVM.h"
+#include "SourceKit/Support/UIdent.h"
+#include "swift/Basic/ThreadSafeRefCounted.h"
+#include "llvm/ADT/ArrayRef.h"
+#include "llvm/ADT/StringRef.h"
+#include "llvm/ADT/IntrusiveRefCntPtr.h"
+
+#include "llvm/Support/ErrorHandling.h"
+#include "llvm/Support/MemoryBuffer.h"
+
+#include <map>
+
+using namespace SourceKit;
+using namespace sourcekitd;
+using llvm::ArrayRef;
+using llvm::StringRef;
+using llvm::raw_ostream;
+
```

Comment:
These using declarations shouldn't be necessary since `"SourceKit/Core/LLVM.h"` is included.